sklearn/base.py

```diff
@@ -39,17 +40,40 @@ def clone(estimator, safe=True):
         The estimator or group of estimators to be cloned
 
     safe : boolean, optional
-        If safe is false, clone will fall back to a deep copy on objects
-        that are not estimators.
+
+        If safe is false, clone will fall back to a copy (deep copy or simple
```

Comment:
Wouldn't this be better described as "if True, the argument to clone must be an estimator (i.e. support `get_params`). Otherwise the object will be copied ..."